src/lambda/expression.rs

```diff
@@ -83,58 +71,39 @@ impl Expression {
         Expression::EqualTo(left, right) => Ok(async_graphql::Value::from(
           left.eval(ctx).await? == right.eval(ctx).await?,
         )),
-        Expression::Unsafe(input, operation) => {
-          let input = input.eval(ctx).await?;
+        Expression::Unsafe(_input, operation) => {
```

Comment:
```suggestion
        Expression::Unsafe(operation) => {
```
Remove `input` from unsafe.